assets/js/script.js

```diff
@@ -0,0 +1,103 @@
+// Assignment Code
+const generateBtn = document.querySelector("#generate");
+
+// get the password length function
+const getPasswordLength = () => {
+  const passwordLength = prompt("how long would you like your password to be?");
+  const passwordLengthNum = parseInt(passwordLength, 10);
+  if (passwordLengthNum >= 8 && passwordLengthNum <= 128) {
+    return passwordLengthNum;
+  } else {
+    // This is when the users' input is not right
+    alert("please enter the password again.");
```

Comment:
These could be more descriptive. So if the password was too long you could say 'Too many characters' etc.